Expecto.Tests/Tests.fs

```diff
@@ -184,8 +184,8 @@ let timeouts =
           match testFromType Dummy.thisModuleType.Value with
           | Some (TestList (
                       Seq.Two (
-                          TestLabel("test B", TestList _),
-                          TestLabel("test A", TestList _)))) -> ()
+                          TestLabel("test B", TestList (_, Normal), Normal),
```

Comment:
The `TestLabel` may not need it: `let test = TestCase (test, Normal)`? Here https://github.com/MecuSorin/expecto/commit/290dbf3c967dbdce5c2bf3b2870ab3669025f07b#diff-312b950ecb6e71200586b2408ab6e821R41